src/main/java/quickcarpet/skyblock/mixin/FlowerPotBlockMixin.java

```diff
@@ -0,0 +1,95 @@
+    private void doActivate(BlockState blockState_1, World world_1, BlockPos blockPos_1, PlayerEntity playerEntity_1, Hand hand_1, BlockHitResult blockHitResult_1) {
+        if (flowerPotChunkLoading && !flowerPotChunkLoadingPowered) {
+            ItemStack itemStack_1 = playerEntity_1.getStackInHand(hand_1);
+            Item item_1 = itemStack_1.getItem();
+            Block block_1 = item_1 instanceof BlockItem ? (Block)CONTENT_TO_POTTED.getOrDefault(((BlockItem)item_1).getBlock(), Blocks.AIR) : Blocks.AIR;
+            boolean boolean_1 = block_1 == Blocks.AIR;
+            boolean boolean_2 = this.content == Blocks.AIR;
+            if (boolean_1 != boolean_2) {
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ(), boolean_2);
+                //This works, but will checking if its another chunk be faster?
+                world_1.setChunkForced(blockPos_1.getX()-1, blockPos_1.getZ(), boolean_2);
+                world_1.setChunkForced(blockPos_1.getX()+1, blockPos_1.getZ(), boolean_2);
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()+1, boolean_2);
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()-1, boolean_2);
+            }
+        }
+    }
+    @Inject(method = "onBreak", at = @At(value = "INVOKE",target = "Lnet/minecraft/block/FlowerPotBlock;onBreak(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/entity/player/PlayerEntity;)V"))
+    public void onOnBreak(World world_1, BlockPos blockPos_1, BlockState blockState_1, PlayerEntity playerEntity_1) {
+        if (flowerPotChunkLoading) {
+            world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ(), false);
+            //This works, but will checking if its another chunk be faster?
+            world_1.setChunkForced(blockPos_1.getX()-1, blockPos_1.getZ(), boolean_2);
+            world_1.setChunkForced(blockPos_1.getX()+1, blockPos_1.getZ(), boolean_2);
+            world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()+1, boolean_2);
+            world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()-1, boolean_2);
+        }
+    }
+    
+    @Inject(method = "neighborUpdate", at = @At(value = "INVOKE",target = "Lnet/minecraft/block/FlowerPotBlock;neighborUpdate(Lnet/minecraft/block/BlockState;Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/Block;Lnet/minecraft/util/math/BlockPos;)V"))
+    public void onNeighborUpdate(BlockState blockState_1, World world_1, BlockPos blockPos_1, Block block_1, BlockPos blockPos_2, boolean boolean_1) {
+        if (flowerPotChunkLoading && flowerPotChunkLoadingPowered) {
+            boolean boolean_1 = world_1.isReceivingRedstonePower(blockPos_1);
+            boolean boolean_2 = this.content == Blocks.AIR; //must have flower in pot (you decide if you want this)
+            if (boolean_1 != (Boolean)blockState_1.get(POWERED) && !boolean_2) {
```

Comment:
How is it possible to have a `POWERED` property? `POWERED` is not even in scope here

Reply:
Right, ill check on this.

Reply:
Want it to check if there's a powered block next to it?